dcspy/logitech.py

```diff
@@ -229,15 +235,15 @@ def __repr__(self) -> str:
 
 class G13(KeyboardManager):
     """Logitech`s keyboard with mono LCD."""
-    def __init__(self, parser: ProtocolParser, **kwargs) -> None:
+    def __init__(self, parser: ProtocolParser, socket: socket, **kwargs) -> None:
```

Comment:
This, and other occurrences, are so we have the socket available for when we receive Gkey key press callback events.

Reply:
I get it... anyway we can try skip passing socket here.. not sure if it is possible. Just open ssue.... Mayby this is good point to discuss add in `FontConfing` in `G13` etc

``` python
class G510(KeyboardManager):
    def __init__(self, parser: ProtocolParser, sock: socket, **kwargs) -> None:
        LcdMono.set_fonts(kwargs['fonts'])
        super().__init__(parser, sock, lcd_type=LcdMono)
```